.github/workflows/sentry-source-map-upload.yaml

```diff
@@ -13,6 +13,8 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
The action was failing due to incomplete history: https://github.com/devtron-labs/dashboard/actions/runs/12431913880/job/34710251093